setup.py

```diff
@@ -80,6 +80,10 @@ def package_files(directory):
                 'MAVProxy.modules.lib.MacOS',
                 'MAVProxy.modules.lib.optparse_gui'],
       install_requires=requirements,
+      extras_require={
+        # restserver module
+        'server': ['flask'],
```

Comment:
Gah.  This was supposed to be `restserver` from a technical perspective...

Reply:
I think that extras_require should be not module specific but in a more generic context, in this case, with web and server features, that's why I'm using server as key here.

Reply:
@peterbarker

Reply:
Ping @peterbarker

Reply:
Ping @peterbarker